mdx-models/src/main/java/com/mx/path/model/mdx/accessor/payment/PaymentBaseAccessor.java

```diff
@@ -51,6 +53,52 @@ public PaymentBaseAccessor(AccessorConfiguration configuration) {
     super(configuration);
   }
 
+  /**
+   * Enrollment status
+   *
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Get the Enrollment status")
+  public AccessorResponse<Enrollment> enrollmentStatus() {
+    throw new AccessorMethodNotImplementedException();
+  }
+
+  /**
+   * Update Enrollment status
+   *
+   * @param enrollmentRequest
+   * @return
+   */
+  @GatewayAPI
+  @API(description = "Update the Enrollment status")
+  public AccessorResponse<Enrollment> updateEnrollmentStatus(Enrollment enrollmentRequest) {
```

Comment:
I'd suggest this parameter could be named 'enrollment' instead of 'enrollmentRequest'

Reply:
`objectRequest` seems to be the convention in controllers (I checked AccountController and TransfersController). I would leave this as is.

Reply:
Keeping the same way since we are following it in all other controllers